core/src/main/java/io/opentelemetry/android/OpenTelemetryRumImpl.java

```diff
@@ -15,13 +15,13 @@
 final class OpenTelemetryRumImpl implements OpenTelemetryRum {
 
     private final OpenTelemetrySdk openTelemetrySdk;
-    private final SessionManager sessionManager;
+    private final SessionProvider sessionProvider;
 
     private final ExtendedLogger logger;
 
-    OpenTelemetryRumImpl(OpenTelemetrySdk openTelemetrySdk, SessionManager sessionManager) {
+    OpenTelemetryRumImpl(OpenTelemetrySdk openTelemetrySdk, SessionProvider sessionProvider) {
```

Comment:
I actually very much like that this is loosening the type -- because `SessionManager` isA `SessionProvider`, it should have been this all along!